doc/global_variables.md

```diff
@@ -48,6 +48,18 @@ Not currently used (planned usage in a future version).
 
 Not currently used (planned usage in a future version).
 
+### `admin-enable_ops_genie_integration`
+
+Enables integration with [OpsGenie](www.opsgenie.com) so that ProxySQL will push critical alerts to OpsGenie.
+
+### `admin-ops_geni_key`
+
+OpsGenie apiKey needed to authenticate when posting alerts.
```

Comment:
As a user of the ProxySQL, I don't care if the Proxy POSTs, snailmails or phones in to push the alert to OpsGenie :)